ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessesResourcesSpec.scala

```diff
@@ -47,36 +48,34 @@ class ProcessesResourcesSpec extends FunSuite with ScalatestRouteTest with Match
   private val processName: ProcessName = ProcessName(SampleProcess.process.id)
 
   test("return list of process") {
```

Comment:
I wonder if all these tests should be here? I mean, we have ProcessService, maybe some tests should be on that level? At least let's add some comment now?

Reply:
In fact we should test there permissions and response data for success. You're right we should test business logic in service test.